toasty/simple_interface.py

```diff
@@ -0,0 +1,51 @@
+from toasty import builder, collection, pyramid, multi_tan, multi_wcs

```

Comment:
I tend to think that this functionality should just go in the toplevel `__init__.py` file rather than a new submodule (with a somewhat long name). Logically it makes sense and it will save typing for the commands that we hope that people might actually be typing regularly.